content/en/docs/devops-user-guide/examples/create-multi-cluster-pipeline.md

```diff
@@ -0,0 +1,264 @@
+keywords: 'KubeSphere, Kubernetes, Multi-cluster, Pipeline, DevOps'
+description: 'Learn how to create a multi-cluster pipeline on KubeSphere.'
+linkTitle: "Create a Multi-cluster Pipeline"
+weight: 11440
+---
+
+As cloud providers offer different hosted Kubernetes services, DevOps pipelines have to deal with use cases where multiple Kubernetes clusters are involved.
+
+This tutorial demonstrates how to create a multi-cluster pipeline on KubeSphere.
+
+## Prerequisites
+
+- You need to have three Kubernetes clusters with KubeSphere installed. Choose one cluster as your Host Cluster and the other two as your Member Clusters. For more information about cluster roles, refer to [KubeSphere Federation](../../../multicluster-management/introduction/kubefed-in-kubesphere/).
+- You need to [enable the KubeSphere DevOps system](../../../pluggable-components/devops/) on your Host Cluster.
+- You need to integrate SonarQube into your pipeline. For more information, refer to [Integrate SonarQube into Pipelines](../../how-to-integrate/sonarqube/).
+- You need to create four accounts: `ws-manager`, `ws-admin`, `project-admin`, and `project-regular`, and grant these accounts different roles. For more information, refer to [Create Workspaces, Projects, Accounts and Roles](../../../quick-start/create-workspace-and-project/#step-1-create-an-account).
+
+## User Case Overview
+
+This tutorial uses a demo user case where different deployment environments are isolated through three clusters. See the diagram as below.
+
+![use-case-for-multi-cluster](/images/docs/devops-user-guide/examples/create-multi-cluster-pipeline/use-case-for-multi-cluster.png)
+
+The three clusters are used as development cluster, testing cluster, and production cluster respectively. Once codes get submitted to a Git repository, a pipeline will be triggered to run through the following stages—unit test, code analysis, image building and pushing, and deployment to development cluster. Developers will manage the development cluster for self-testing and validation. When developers give an approval, the pipeline will proceed to the stage of deployment to testing cluster for stricter validation. Finally, the pipeline, with necessary approval ready, will reach the stage of deployment to production cluster to provide services externally. 
+
+## Hands-on Lab
+
+### Step 1: Prepare clusters
+
+This tutorial uses three clusters as described in the table below. You can log in to the cluster to be used as the Host Cluster as `admin` to set up your multi-cluster environment by referring to [Direct Connection](../../../multicluster-management/enable-multicluster/direct-connection/) or [Agent Connection](../../../multicluster-management/enable-multicluster/agent-connection/). Make sure you set the Member clusters as [public clusters](../../../cluster-administration/cluster-settings/cluster-visibility-and-authorization/#make-a-cluster-public). The below operations will be carried out on the Host Cluster.
```

Comment:
This paragraph is a little bit redundant as you already mentioned that users need to have three clusters in Prerequisites. Change the whole paragraph to:
->
See the table below for the role of each cluster.